MK2CHOPR/MK2CHOPR-IOC-01App/Db/MK2CHOPR.db

```diff
@@ -0,0 +1,391 @@
+    info(INTEREST, "HIGH")
+    info(archive, "VAL")
+}
+
+record(ai, "$(P)SIM:PHASE")
+{
+}
+
+alias("$(P)SIM:PHASE","$(P)SIM:PHASE:SP")
+alias("$(P)SIM:PHASE","$(P)SIM:PHASE:SP:RBV")
+
+
+#######################################################
+# Phase errors
+#######################################################
+
+record(ai, "$(P)PHASE:ERR")
+{
+    field(DESC, "Actual phase delay error")
+    field(DTYP, "stream")
+    field(INP, "@mk2chopper.proto getTruePhaseErr $(PORT)")
+    field(SCAN, "1 second")
+    field(EGU, "us")
+    field(PREC, "0")
+    field(SIML, "$(P)SIM")
+    field(SIOL, "$(P)SIM:PHASE:ERR")
+    field(SDIS, "$(P)DISABLE")
+    info(INTEREST, "HIGH")
+    info(archive, "VAL")
+}
+
+record(ao, "$(P)PHASE:ERR:SP")
+{
+    field(DESC, "Set the demanded PD error window")
+    field(DTYP, "stream")
+    field(OUT,  "@mk2chopper.proto setPhaseErr $(PORT)")
+    field(EGU, "us")
+    field(PREC, "0")
+    field(SIML, "$(P)SIM")
+    field(SIOL, "$(P)SIM:PHASE:ERR:SP")
+    field(SDIS, "$(P)DISABLE")
+}
+
+record(ai, "$(P)PHASE:ERR:SP:RBV")
+{
+    field(DESC, "Readback of demanded PD error window")
+    field(DTYP, "stream")
+    field(INP, "@mk2chopper.proto getDemPhaseErr $(PORT)")
+    field(SCAN, "1 second")
+    field(EGU, "us")
+    field(PREC, "0")
+    field(SIML, "$(P)SIM")
+    field(SIOL, "$(P)SIM:PHASE:ERR")
+    field(SDIS, "$(P)DISABLE")
+    info(INTEREST, "HIGH")
+    info(archive, "VAL")
+}
+
+record(ai, "$(P)SIM:PHASE:ERR")
+{
+}
+
+alias("$(P)SIM:PHASE:ERR","$(P)SIM:PHASE:ERR:SP")
+alias("$(P)SIM:PHASE:ERR","$(P)SIM:PHASE:ERR:SP:RBV")
+
+#######################################################
+# Start/stop
+#######################################################
+
+record(bo, "$(P)ON:SP")
+{
+    field(DESC, "Set the chopper off(0)/on(1)")
+    field(DTYP, "stream")
+    field(OUT,  "@mk2chopper.proto setChopperStarted $(PORT)")
+    field(EGU, "")
+    field(SIML, "$(P)SIM")
+    field(SIOL, "$(P)SIM:ON:SP")
+    field(SDIS, "$(P)DISABLE")
+}
+
+record(calcout, "$(P)ON:CALC") {
+  field(INPA, "$(P)FREQ")
+  field(CALC, "A>0?1:0")
+  field(DOPT, "Use CALC")
+  field(OOPT, "Every Time")
+  field(OUT, "$(P)ON PP")
+}
+
+record(bi, "$(P)ON")
+{
+  field(DESC, "Chopper is on/off")
+  field(EGU, "")
+  field(ZNAM, "Off")
+  field(ONAM, "On")
+  field(SDIS, "$(P)DISABLE")
+}
+
+record(bo, "$(P)SIM:ON")
+{
+}
+
+alias("$(P)ON","$(P)ON:SP:RBV")
+alias("$(P)SIM:ON","$(P)SIM:ON:SP")
+alias("$(P)SIM:ON","$(P)SIM:ON:SP:RBV")
+
+#######################################################
+# Chopper locks
+#######################################################
+
+record(mbbiDirect, "$(P)CHOP:STAT") {
+  field(DESC, "Chopper interlock status")
+  field(SDIS, "$(P):DISABLE")
+  field(SCAN, "1 second")  
+  field(DTYP, "stream")
+  field(INP, "@mk2chopper.proto getChopperLocks $(PORT)")
+  field(SIML, "$(P)SIM")
+  field(SIOL, "$(P)SIM:CHOP:STAT")
+}
+
+record(mbbiDirect, "$(P)SIM:CHOP:STAT")
+{
+}
```

Comment:
Should this record have a `DESC` field at least (if it doesn't need anything else?)